### ISSUE TITLE
Error in recent_articles.html while generating public site

### ISSUE THREAD

**Issue by zeebluejay** [digitalcraftsman/hugo-icarus-theme#32]
Hello,

I just installed Hugo Static Site Generator v0.15 BuildDate: 2015-11-26T01:29:07-05:00 on my Macbook and got the Icarus theme. Followed all the steps outlined in the theme installation [link](http://themes.gohugo.io/hugo-icarus/)

I then created a new post with hugo command. Edited it to add some text.

But I get errors when to execute:

```
$ hugo --verbose server
```

These errors prevent the static site to be generated. I don't understand what I did wrong.
Please help.

Thanks!

Here are the errors:

```
$ hugo --verbose server
INFO: 2016/05/14 Using config file: /Users/zeekay/dev/hugo-blog/mynewblog/config.toml
INFO: 2016/05/14 syncing from /Users/zeekay/dev/hugo-blog/mynewblog/themes/hugo-icarus-theme/static to /
INFO: 2016/05/14 syncing from /Users/zeekay/dev/hugo-blog/mynewblog/static/ to /
INFO: 2016/05/14 found taxonomies: map[string]string{"tag":"tags", "category":"categories"}
ERROR: 2016/05/14 template: theme/partials/widgets/recent_articles.html:26:97: executing "theme/partials/widgets/recent_articles.html" at <index .Params.catego...>: error calling index: index out of range: 0 in theme/partials/widgets/recent_articles.html
ERROR: 2016/05/14 template: theme/partials/widgets/recent_articles.html:26:97: executing "theme/partials/widgets/recent_articles.html" at <index .Params.catego...>: error calling index: index out of range: 0 in theme/partials/widgets/recent_articles.html
ERROR: 2016/05/14 template: theme/partials/widgets/recent_articles.html:26:97: executing "theme/partials/widgets/recent_articles.html" at <index .Params.catego...>: error calling index: index out of range: 0 in theme/partials/widgets/recent_articles.html
INFO: 2016/05/14 Alias "/post/page/1" translated to "post/page/1/index.html"
ERROR: 2016/05/14 template: theme/partials/widgets/recent_articles.html:26:97: executing "theme/partials/widgets/recent_articles.html" at <index .Params.catego...>: error calling index: index out of range: 0 in theme/partials/widgets/recent_articles.html
ERROR: 2016/05/14 template: theme/partials/widgets/recent_articles.html:26:97: executing "theme/partials/widgets/recent_articles.html" at <index .Params.catego...>: error calling index: index out of range: 0 in theme/partials/widgets/recent_articles.html
INFO: 2016/05/14 Alias "/page/1" translated to "page/1/index.html"
ERROR: 2016/05/14 template: theme/partials/widgets/recent_articles.html:26:97: executing "theme/partials/widgets/recent_articles.html" at <index .Params.catego...>: error calling index: index out of range: 0 in theme/partials/widgets/recent_articles.html
0 draft content
0 future content
1 pages created
2 paginator pages created
0 tags created
0 categories created
in 27 ms
Watching for changes in /Users/zeekay/dev/hugo-blog/mynewblog/{data,content,layouts,static,themes}
Serving pages from memory
Web Server is available at http://localhost:1313/ (bind address 127.0.0.1)
```

**Comment by digitalcraftsman** [comment]
Hello @zeebluejay,

excuse the delay. The theme tries to access a non-existing category name from the frontmatter of a content file. It could be fixed with eda9d564550a4db678449b61b59728601befa9cb. Could you fetch the latest version of this theme and perhaps the new release of Hugo (v0.16) in order to check if the error is still present?

Cheers,
Digitalcraftsman

**Comment by digitalcraftsman** [comment]
Closing this as stall.
